File: java/src/main/java/com/languageweaver/sdk/samples/lps/EdgeLanguagePairService.java
package com.languageweaver.sdk.samples.lps;

import com.languageweaver.sdk.common.edge.EdgeLanguageWeaverClient;
import com.languageweaver.sdk.translate.edge.result.EdgeLanguagePairsResult;

public class EdgeLanguagePairService {

    public static void main(String[] args) throws Exception {
        try (EdgeLanguageWeaverClient lwClient = new EdgeLanguageWeaverClient().build()) {
            final EdgeLanguagePairsResult languagePairsResult = lwClient.getEdgeLanguagePairs();

            if (languagePairsResult.getLanguagePairs() != null) {
                languagePairsResult.getLanguagePairs()
                        .forEach(languagePair -> System.out.println(languagePair.getLanguagePairId()));
            }
        }
    }

}

File: dotnet/Samples/Lps/EdgeLanguagePairService.cs
﻿using LanguageWeaver.Sdk.Common.Edge;
using LanguageWeaver.Sdk.LanguagePairs.Edge.Result;

namespace LanguageWeaver.Sdk.Samples.Lps;

public static class EdgeLanguagePairService
{
    public static void Main()
    {
        using var lwClient = new EdgeLanguageWeaverClient().Build();
        EdgeLanguagePairsResult languagePairsResult = lwClient.GetEdgeLanguagePairs();

        languagePairsResult.LanguagePairs
            .ForEach(languagePair => Console.WriteLine(languagePair.LanguagePairId));
    }
}